### ISSUE TITLE
Default to date/time if filename left blank

### ISSUE THREAD

**Issue by fennectech** [J-D-K/JKSV#1]
We should allow users to simply hit start on the save keyboard without adding any text and just use the current date and time.

**Comment by J-D-K** [comment]
Pressing R will auto fill it with the date since it's based on the pre-swkbd 3DS keyboard.

**Comment by J-D-K** [comment]
I've also just added the old 3DS shortcuts in the most recent commit (Holding L or R while selecting New) if that's good enough? The string being empty is what is checked before creating a folder and backing up the saves.

**Comment by fennectech** [comment]
Works for me :D.   I use JKSV for minecraft because checkpoint screams in pain about bedrock saves.       Its a nice project :D

**Comment by J-D-K** [comment]
I just _hopefully_ got the UTF8 stuff straightened out. I thought I read that Switch supported UTF8, but apparently there's a bug in Horizon?

I can only speak for myself, but what was messing JKSV up was that Bedrock paths have no spaces in them and the paths are pretty long. My text wrapping searched for spaces to break the text into lines and would crash just like that. I added '/' to the break characters and that fixed it. Not ideal, but it worked.

**Comment by fennectech** [comment]
Nope. Switch supports utf8 but fat32 does not.    You cant put Unicode into filenames on fat32 filesystems
